includes/class-wc-gateway-ppec-client.php

```diff
@@ -458,72 +458,7 @@ protected function _get_details_from_cart() {
 			'items'             => $this->_get_paypal_line_items_from_cart(),
 		);
```

Comment:
nvm. Currently, it's confusing already because we have `_get_details_from_cart` and `_get_details_from_order`.